src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+.osdQueryEditor__wrap {
+  height: 200px;
+  width: 500px;
+}
+
+.osdQueryEditorHeader {
+  max-height: 400px;
+
+  // TODO fix styling: with "overflow: auto" the scroll bar appears although the content is below max-height
+  // overflow: auto;
+}
+
+@include euiBreakpoint("xs", "s") {
+  .osdQueryEditor--withDatePicker {
+    > :first-child {
+      // Change the order of the query bar and date picker so that the date picker is top
+      // and the query bar still aligns with filters
+      order: 1;
+
+      // EUI Flexbox adds too much margin between responded items, this just moves it up
+      margin-top: $euiSizeS * -1;
+    }
+  }
+}
+
+// IE specific fix for the datepicker to not collapse
+@include euiBreakpoint("m", "l", "xl") {
+  .osdQueryEditor__datePickerWrapper {
+    max-width: 40vw;
+    flex-grow: 0 !important;
+    flex-basis: auto !important;
+
+    &.osdQueryEditor__datePickerWrapper-isHidden {
```

Comment:
Great point. This is also ported over but you are right. There lacks a standardizes this. In general, there isn't an strict rules regarding styling and class names.

It's something that we noticed and I believe a standardization would have been apart of pink elephant project.

We started some of that work here: https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/packages/osd-stylelint-config/config

And we could add a rule for convention and we could start running it apart of the a CI but it would cause CI's for plugins to fail.

Will note here: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6957